docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -347,7 +347,8 @@ def docker_registry_client
           DockerRegistry2::Registry.new(
             "https://#{registry_hostname}",
             user: registry_credentials&.fetch("username", nil),
-            password: registry_credentials&.fetch("password", nil)
+            password: registry_credentials&.fetch("password", nil),
+            read_timeout: 10,
```

Comment:
```suggestion
            read_timeout: 10
```

To appease the RuboCop

Reply:
🚲 shed: 
I'd also be open to disabling this rubocop instead... I'm personally a fan of trailing commas because it preserves `git blame` history/smaller diffs if someone later appends another method arg to the list. Yes, I realize this is a [contentious issue](https://github.com/rubocop/ruby-style-guide/issues/273), and no, it's not something I feel strongly about.

Reply:
I'm fine either way, but I kind of like being able to run `rubocop -A` and have it produce something consistent 🤷